packages/adblocker-webextension/src/index.ts

```diff
@@ -564,13 +656,12 @@ export class WebExtensionBlocker extends FiltersEngine {
   };
 
   public onHeadersReceived = (
-    _: Browser,
+    browser: Browser,
     details: OnHeadersReceivedDetailsType,
   ): WebRequest.BlockingResponse => {
-    return updateResponseHeadersWithCSP(
-      details,
-      this.getCSPDirectives(fromWebRequestDetails(details)),
-    );
+    const request = fromWebRequestDetails(details);
```

Comment:
Do we need to specify the generic type `T` here?

Reply:
from what I see it is being inferred from details